modules/nixos-wiki/default.nix

```diff
@@ -177,6 +177,9 @@ in
 
         # Block spam by regex
         $wgSpamRegex = ["/seo (software|tools)|adult toys|pornstars|casino|gambling|viagra/i"];
+
+        # Enable String Parser functions
+        $wgEnableStringFunctions = true;
```

Comment:
Should we go for this or the other option mentioned in this wiki? https://www.mediawiki.org/wiki/Extension:ParserFunctions/String_functions